plugins/chat/app/controllers/chat/api/chatables_controller.rb

```diff
@@ -5,17 +5,103 @@ def index
     params.require(:filter)
     filter = params[:filter].downcase
 
+    users = filtered_users(filter: filter)
```

Comment:
ultimately would be nice if all of this could be done using a chat service, but we can do this at a later time if you don't want to dig this now